src/docfx/build/metadata/SystemMetadata.cs

```diff
@@ -61,4 +61,7 @@ internal class SystemMetadata
 
     [JsonProperty("_op_pdfUrlPrefixTemplate")]
     public string? PdfUrlPrefixTemplate { get; set; }
+
+    [JsonProperty("xrefs")] // the xrefs defined in this file
+    public List<ExternalXrefSpec>? Xrefs { get; set; }
```

Comment:
If we'd like export xref as a string metadata due to whatever reason then let's define it as `string` consistently, it is better than having some places as strings and some places as an array.

```suggestion
    public string? Xrefs { get; set; }
```

Reply:
Good point!
Just update the PR.

Reply:
Just revert the change.

The SystemMetadata is output in structured json and passed to Localization Transformer in Xliff Build.

Localization Transformer go through the json element tree and try to figure out which properties should be localized via a allowlist (e.g. In xrefs, name, summary, description should be localized)

Therefore, the xrefs should be json type instead of string type.

Otherwise, Juno content team will have to translate the whole json string.